optaplanner-core/src/main/java/org/optaplanner/core/config/domain/ReflectionsWorkaroundClasspathHelper.java

```diff
@@ -203,7 +203,7 @@ public static URL forClass(Class<?> aClass, ClassLoader... classLoaders) {
                 if (classLoader instanceof URLClassLoader) {
                     URL[] urls = ((URLClassLoader) classLoader).getURLs();
                     if (urls != null) {
-                        result.addAll(Sets.<URL>newHashSet(urls));
+                        result.addAll(Sets.<URL> newHashSet(urls));
```

Comment:
Not nice. And it makes Checkstyle's [GenericWhitespace](https://checkstyle.sourceforge.io/config_whitespace.html#GenericWhitespace) rule fail.

Reply:
I think that the explicit type argument itself is not nice - since Java 8, type inference is able to handle it [1]. I would rather remove it completely.

[1] https://docs.oracle.com/javase/tutorial/java/generics/genTypeInference.html